x-translator/smartapi_x-translator_schema.json

```diff
@@ -1,9 +1,15 @@
 {
   "title": "x-translator extension within info.",
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
-  "description": "x-translator is a JSON object describes API-level, Translator-specific information. It is an extension/additional property inside the OpenAPI Info Object. Currently only one component-type allowed per API",
+  "description": "x-translator is a JSON object describes API-level, Translator-specific information. It is an extension/additional property inside the OpenAPI Info Object. Currently only one component-type allowed per API. x-translator should be present when the translator tag is present, and vice versa.",
   "properties": {
+    "tags": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/tag"
+      }
+    },
```

Comment:
seems removing this part completely works too?

Reply:
This was copied from the SmartAPI schema too. 

I agree that this could be removed completely (the if-then still catches issues). However, the tags sub-schema can be used to constrain what goes into the tags. For example: in the latest commit, the sub-schema will throw an validation error if the tag isn't a string (- "name": 9606). 

Let me know what you think. I can remove the tags sub-schema completely if needed.

Reply:
Got it. We don't need to include any validations already handled in OpenAPI and SmartAPI schemas. The extensions schema here will be validated only if the input metadata passes the generic OpenAPI schema, so you can assume the metadata is already a valid OpenAPI metadata.

Reply:
Okay, the newest commit removes the tags sub-schema completely (it sounds like validation for that is already handled in OpenAPI or SmartAPI schemas). 

This likely addresses both comments that were made earlier.